src/ComputeClosestToZeroTemperature/index.ts

```diff
@@ -0,0 +1,32 @@
+export class ComputeClosestToZeroTemperature {
+  private convertNegativeToPositiveValue(value: number): number {
+    return value < 0 ? value * -1 : value;
+  }
+  static check(temperatures: number[]): number {
+    if (!temperatures || temperatures.length < 1) return 0;
```

Comment:
```suggestion
    if (!temperatures?.length) return 0;
```

Reply:
will have test coverage issue due to optional value
```suggestion
    if (!temperatures.length) return 0;
```